Assets/Scripts/Player/PlayerController.cs

```diff
@@ -678,6 +678,10 @@ public class PlayerController : Entity {
 		if (dashTimeout != null) StopCoroutine(dashTimeout);
 	    dashTimeout = StartCoroutine(RunDashCooldown(dashCooldownLength));
 	}
+	public void reduceDashCooldown()
```

Comment:
Class methods in C# should start with an uppercase letter btw

Reply:
Oh ok.